docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -12,25 +12,29 @@ This topic walks you through the steps to backup and restore Cassandra data runn
 * K8ssandra Helm chart, which we'll extend with `backupRestore` Medusa buckets for Amazon S3 integration
 * Sample files in GitHub:
   * [medusa-bucket-key.yaml](medusa-bucket-key.yaml) to create a secret with credentials for AWS S3 buckets
-  * [backup-restore-values.yaml](backup-restore-values.yaml) to enable Medusa (backup/restore service) and set related minimal values
+  * [backup-restore-values.yaml](backup-restore-values.yaml) to enable Medusa (backup/restore service) and set values
   * [test_data.cql](test_data.cql) to populate a Cassandra keyspace and table with data
-
+ 
 ## Prerequisites
 
 * A Kubernetes environment
 * Storage for the backups - see below
 * [Helm](https://helm.sh/), a packaging manager for Kubernetes
-* An edited version of [medusa-bucket-key.yaml](./medusa-bucket-key.yaml), as noted below
 
-All other prerequisites are handled by the installed tools listed above. The sample files are checked into GitHub.
+All other prerequisites are handled by the installed tools listed above. 
 
 ## Steps
 
 ### Verify you've met the prereqs
 
 You will need storage for the backups. This topic shows the use of AWS S3 buckets.
 
-* If you'll use AWS S3, before proceeding with the configuration described below, verify that you know the `aws_access_key_id` and `aws_secret_access_key` values. Or  contact your IT team if they manage those assets. You'll provide those details in an edited version of the [medusa-bucket-key.yaml](medusa-bucket-key.yaml) file. For information about the S3 setup steps, see this helpful [readme](https://github.com/thelastpickle/cassandra-medusa/blob/master/docs/aws_s3_setup.md).  
+* If you'll use AWS S3, before proceeding with the configuration described below, verify that you know:
+  * The `aws_access_key_id` and `aws_secret_access_key` values
+  * The `name` of the S3 bucket
+  * The region assigned to the S3 bucket
```

Comment:
https://github.com/k8ssandra/k8ssandra/pull/317 has changes to allow using different regions for the S3 bucket. I hope to get it merged tomorrow. We will want to add docs around the new properties that are introduced in the PR.

Reply:
Ok sounds good - thanks John. We might as well wait for that code merge before proceeding with this doc merge.

Reply:
Removed note and text about matching region for S3